pkg/cni/plugin/plugin.go

```diff
@@ -39,11 +41,13 @@ import (
 
 	"kmesh.net/kmesh/pkg/logger"
 	"kmesh.net/kmesh/pkg/utils"
+	"kmesh.net/kmesh/pkg/utils/hash"
 )
 
 var (
-	log               = logger.NewLoggerFieldWithoutStdout("plugin/cniplugin")
-	ENABLE_KMESH_MARK = "0x1000"
+	log                        = logger.NewLoggerFieldWithoutStdout("plugin/cniplugin")
+	ENABLE_KMESH_MARK          = "0x1000"
+	ENABLED_KMESH_MAP_PIN_PATH = "/sys/fs/bpf/bpf_kmesh_workload/map/map_of_kmesh_manager"
```

Comment:
can this be more explicit?

Reply:
This path is the pinned address of an ebpf map. This map is used to record the IP addresses that are managed by kmesh. After obtaining the FD of the map from the pinned path, we can directly perform operations on the map.